generators/php/model/src/object/ObjectGenerator.ts

```diff
@@ -23,24 +23,17 @@ export class ObjectGenerator extends FileGenerator<PhpFile, ModelCustomConfigSch
         const clazz = php.dataClass({
             ...this.classReference,
             docs: this.typeDeclaration.docs,
-            parentClassReference: this.context.getSerializableTypeClassReference()
+            parentClassReference: this.context.getSerializableTypeClassReference(),
+            usedTraits: this.objectDeclaration.extends.map((declaredTypeName) =>
+                this.context.phpTypeMapper.convertToTraitClassReference(declaredTypeName)
+            )
         });
 
-        // TODO: handle extended properties
         for (const property of this.objectDeclaration.properties) {
-            const convertedType = this.context.phpTypeMapper.convert({ reference: property.valueType });
-            clazz.addField(
-                php.field({
-                    type: convertedType,
-                    name: this.context.getPropertyName(property.name.name),
-                    access: "public",
-                    docs: property.docs,
-                    attributes: this.context.phpAttributeMapper.convert({
-                        type: convertedType,
-                        property
-                    })
-                })
-            );
+            clazz.addField(this.toField({ property, inherited: false }));
```

Comment:
nit: We can drop the `inherited: false` parameter if we make the parameter optional in `toField`.

Reply:
nice, done!